buildingspy/tests/test_development_check_parameters.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import unittest
+
+class Test_development_check_parameters(unittest.TestCase):
+    """
+       This class contains the unit tests for
+       :mod:`buildingspy.development.check_parameters.
+    """
+
+    def test_validateModelParameters(self):
+        import os
+        import buildingspy.development.check_parameters as v
+        val = v.Validator()
+        myMoLib = os.path.join("buildingspy", "tests", "MyModelicaLibrary")
+        val.validateModelParameters(myMoLib)
+
```

Comment:
Add also more tests, for example, add tests that verify that you catch wrong entries. The test you have now only shows that a correct formatted library passes, but it does not verify that a wrong entry is catch. (You may need to refactor your code a bit to test whether wrong entries are caught.)